lib/service/map_direction.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert';
+
+import 'package:concordi_around/model/direction.dart';
+import 'package:concordi_around/model/list_item.dart';
+import 'package:dio/dio.dart';
+import 'package:uuid/uuid.dart';
+
+import '../credential.dart';
+
+class MapDirection {
+
+  var uuid = Uuid();
+  String _sessionToken;
+
+    // Google Directions API requests
+  Future<List<ListItem>> _getDirection(String origin, String destination) async {
+    if (origin.isEmpty || destination.isEmpty) {
+      throw("Origin or destination is empty");
+    }
+    if (_sessionToken == null) {
+      _sessionToken = uuid.v4();
+    }
+    
+    String baseURL = 'https://maps.googleapis.com/maps/api/directions/json';
+
+    String request = '$baseURL?origin=$origin&destination=$destination&key=$PLACES_API_KEY&sessiontoken=$_sessionToken';
```

Comment:
PLACES_API_KEY should be replaced with DIRECTIONS_API_KEY

Reply:
Also remove sessiontoken parameter here as per above

Reply:
Add the [mode](https://developers.google.com/maps/documentation/directions/intro#TravelModes) parameter

Reply:
Places and directions seem to be sharing one API key

Reply:
Mode parameter is pending, changes from #101 is needed

Reply:
The way that the credentials.dart file was done is that. PLACES_API_KEY can differ from DIRECTIONS_API_KEY.

mode should still be part of the API request regardless of #101